timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesApp.scala

```diff
@@ -379,6 +381,31 @@ private[timeseries] trait TimeSeriesApp { self: TimeSeriesScheduler =>
       else
         Ok(getCalendar())
 
+    case GET at url"/api/timeseries/lastruns?job=$jobId" =>
+
+      def toFiniteInstant(bound: Bound[Instant]): Instant = bound match {
```

Comment:
Would be better to handle the fact that a Bound is Finite or not before the serialization phase. The api call will return a 500 with "expected finite instant", which might not be self-explanatory.